ush/ioda/bufr2ioda/marine/b2i/bufr2ioda_insitu_profile_pirata.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+
+import sys
+from b2iconverter.util import parse_arguments
+from b2iconverter.bufr2ioda_config import Bufr2iodaConfig
+from b2iconverter.bufr2ioda_converter import Bufr2ioda_Converter
+from mbuoyb_tropical_ioda_variables import MbuoybTropicalIODAVariables
+from wmo_codes import *
+
+import numpy as np
+
+
+platform_description = 'PIRATA Tropical mooring profiles from mbuoyb: temperature and salinity'
+
+
+class PirataConfig(Bufr2iodaConfig):
+    def ioda_filename(self):
+        return f"{self.cycle_type}.t{self.hh}z.insitu_profile_pirata.{self.cycle_datetime}.nc4"
```

Comment:
Should we start outputing "nc" files instead of "nc4". I foresee consistency issues in the jcb-base yaml if we mix the two. @guillaumevernieres

Reply:
not here @JohnSteffen-NOAA , but something to remember when @givelberg PR's in obsForge.

Reply:
@JohnSteffen-NOAA  and @guillaumevernieres , Is the code actually generating .nc or .nc4 files ?
(.nc files are commonly associated with the NetCDF Classic format (NetCDF-3). This is the older and more widely compatible format.
.nc4 files typically indicate the NetCDF-4 format (which uses HDF5 as its storage layer). NetCDF-4 introduces several advancements)
It may not matter to us, but we may want to run this by NCO or EIB.